types/resize-observer-browser/index.d.ts

```diff
@@ -1,25 +1,23 @@
 // Type definitions for non-npm package resize-observer-browser 0.1
 // Project: https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver, https://developers.google.com/web/updates/2016/10/resizeobserver, https://wicg.github.io/ResizeObserver/
-// Definitions by: Chives <https://github.com/chivesrs>
+// Definitions by: Chives <https://github.com/chivesrs>, William Furr <https://github.com/wffurr>
```

Comment:
Could you use the multline syntax here please?
https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package

Reply:
Done.